madminer/core/madminer.py

```diff
@@ -668,6 +668,7 @@ def run(
         initial_command=None,
         python2_override=False,
         systematics=None,
+		order='LO',
     ):
 
         """
```

Comment:
Code changes look great. The new `order` keyword is not yet documented in the docstring.

Reply:
I've added some documentation in the docstring